src/lib/code-sample/go.ts

```diff
@@ -0,0 +1,167 @@
+import { pascalCase } from 'change-case'
+
+import type { Json, NonNullJson } from 'lib/json.js'
+
+import { createJsonResponse } from './create-json-response.js'
+import type { CodeSampleDefinition, Context } from './schema.js'
+
+const DEFAULT_GO_PACKAGE_NAME = 'api'
+const GO_PACKAGE_BASE_PATH = 'github.com/seamapi/go'
+
+export const createGoRequest = (
+  { request }: CodeSampleDefinition,
+  _context: Context,
+): string => {
+  const isReqWithParams = Object.keys(request.parameters).length !== 0
+  const goPackageName = getGoPackageName(request.path)
+
+  const goSdkImports = generateImports({
+    goPackageName,
+    isReqWithParams,
+  })
+
+  const requestStructName = getRequestStructName(request.path)
+  const formattedArgs = formatGoArgs(request.parameters)
+  const goSdkRequestArgs = `context.Background()${isReqWithParams ? `, ${goPackageName}.${requestStructName}(${formattedArgs})` : ''}`
+
+  const pathParts = request.path.split('/')
+
+  return `${goSdkImports}
+
+  client${pathParts.map((p) => pascalCase(p)).join('.')}(${goSdkRequestArgs})
+  `.trim()
+}
+
+const getGoPackageName = (path: string): string => {
```

Comment:
What is an example where the import is not `import api "github.com/seamapi/go`?

Reply:
For ex., `import acs "github.com/seamapi/go/acs` for `acs.UsersListAccessibleEntrancesRequest` or 
`import accesscodes "github.com/seamapi/go/accesscodes` for `accesscodes.UnmanagedConvertToManagedRequest` or
`import useridentities "github.com/seamapi/go/useridentities` for `useridentities`EnrollmentAutomationsDeleteRequest` etc.